src/chromeDebugAdapter.ts

```diff
@@ -347,6 +332,37 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         this._chromeProc = null;
     }
 
+    private async killChromeOnWindows(chromePID: number): Promise<void> {
```

Comment:
Do we have enough telemetry to figure it if this method/logic is working as expected or not?